packages/js-drive/lib/createDIContainer.js

```diff
@@ -781,12 +787,35 @@ function createDIContainer(options) {
 
     infoHandler: asFunction(infoHandlerFactory).singleton(),
     checkTxHandler: asFunction(checkTxHandlerFactory).singleton(),
-    beginBlockHandler: asFunction(beginBlockHandlerFactory).singleton(),
-    deliverTxHandler: asFunction(deliverTxHandlerFactory).singleton(),
+
+    beginBlockHandler: asFunction(beginBlockFactory).singleton(),
+    beginBlock: asFunction((
+      enrichErrorWithConsensusError,
+      beginBlockHandler,
+    ) => enrichErrorWithConsensusError(beginBlockHandler)).singleton(),
+    commitHandler: asFunction(commitFactory).singleton(),
+    commit: asFunction((
+      enrichErrorWithConsensusError,
+      commitHandler,
+    ) => enrichErrorWithConsensusError(commitHandler)).singleton(),
+    deliverTxHandler: asFunction(deliverTxFactory).singleton(),
+    deliverTx: asFunction((
+      wrapInErrorHandler,
+      enrichErrorWithConsensusError,
+      deliverTxHandler,
+    ) => wrapInErrorHandler(enrichErrorWithConsensusError(deliverTxHandler))).singleton(),
```

Comment:
We shouldn't throw internal error. To do so please use option `respondWithInternalError: true`